prov/usnic/src/usdf_ep_msg.c

```diff
@@ -347,13 +308,19 @@ int usdf_msg_fill_tx_attr(uint32_t version, struct fi_info *hints,
 		return -FI_ENODATA;
 
 out:
+	/* catch version changes here. */
```

Comment:
Nit: the label `out` doesn't seem like an appropriate name any more -- because you goto `out` when you're just going to go *out*.  But here, you're not necessarily just going out -- you're still doing some checks, and *then* you go "out".

The same is true in other places, too.